compute/snapshots.go

```diff
@@ -99,3 +99,59 @@ func (c *SnapshotsClient) Delete(ctx context.Context, input *DeleteSnapshotInput
 
 	return nil
 }
+
+type StartMachineFromSnapshotInput struct {
+	MachineID string
+	Name      string
+}
+
+func (c *SnapshotsClient) StartMachineFromSnapshot(ctx context.Context, input *StartMachineFromSnapshotInput) error {
```

Comment:
From chat it sounds like we'll change this to `compute.Snapshots().StartMachine()`.